src/utils/FormInputHandling.ts

```diff
@@ -17,14 +17,65 @@ async function getData(currentPage: number, limit: number) {
 
 // post data to the server
 async function postFormData(formData: RowType) {
-  const currentDate = new Date().toLocaleDateString();
+  const { student, test, timeline, dateCreated } = formData;
+  let start_time = timeline.startDate + "T" + timeline.startTime + ":00";
```

Comment:
str(datetime.strptime(row["start_date"] + " " + row["start_time"], date_format)),

date_format = "%Y-%m-%d %I:%M:%S %p"

this is what i do with time in python .. can u verify if this is what is happening in JS too..

Reply:
similar but I've now updated it with `formatDateTime` function in `FormInputHandling.ts ` that does this